### DISCUSSION TITLE
Fine mesh encountered problems

### DISCUSSION THREAD

**Discussion by hzw-zju-2023** [AMReX-Combustion/PeleC#794] [Q&A]
I'm simulating a jet flame. So I refined the grid to 3 levels. What I found, however, was that when I looked at the results with paraview. The most refined areas cannot be displayed. I have tried static fine mesh and dynamic fine mesh and have encountered this problem. When I reduced the level of refinement displayed by paraview, I could display all the areas, but I couldn't accept the result. I don't know what's wrong with my Settings. I have attached input files for my dynamic refinement and static refinement grids. Please take a look.
[input.zip](https://github.com/AMReX-Combustion/PeleC/files/15213038/input.zip)
![d](https://github.com/AMReX-Combustion/PeleC/assets/154048455/9c3ec490-fbe5-4ad7-828d-408eb9c3cb84)
![s](https://github.com/AMReX-Combustion/PeleC/assets/154048455/d9f5a098-aabf-48c8-9800-2c5d1f6ec901)

**Comment by KaisenseChen** [comment]
谢谢你的来信，我会尽快回复你的。

**Comment by marchdf** [comment]
Maybe there are too many cells on the fine levels? I would recommend trying another visualizer like visit and see if the problem persists. Or maybe even amrvis. We link to those in the readme.
